config/webpack.styles.config.js

```diff
@@ -74,6 +74,9 @@ const scssLoaders = Object.assign(
         {
           loader: 'sass-loader',
           options: {
+            sourceMap: true,
```

Comment:
I'm seeing a bunch of logs to the effect of "Previous source map found, but options.sourceMap isn't set" from PostCSS Loader. Not sure if this is related?

Reply:
Not sure, are you seeing this in compile time?

Reply:
It seems to be at runtime when I load the catalog.